dissect/cim/classes.py

```diff
@@ -331,6 +331,8 @@ def key(self) -> InstanceKey:
 
 
 class InstanceKey(dict):
+    """InstanceKey"""
```

Comment:
```suggestion
    """InstanceKey."""
```

Reply:
More than one word please.

Reply:
```suggestion
    """A dictionary subclass that allows for attribute-style access and assignment.

    Is represented as a string in the format "key1=value1;key2=value2" or "default" if empty.

    Example:

        >>> d = InstanceKey()
        >>> d.foo = "bar"
        >>> d.foo
        "bar"
    """
```

Edit, how about this?